x/vote/exported/types.go

```diff
@@ -64,6 +64,10 @@ func (m PollMetadata) Validate() error {
 		return fmt.Errorf("invalid min voter count")
 	}
 
+	if len(m.Voters) == 0 {
```

Comment:
Can we add a check that `MinVoterCount` is greater than 0 in the evm param validation?